package.json

```diff
@@ -9,9 +9,9 @@
     "format": "prettier --write .",
     "lint": "eslint packages/*/{src,types}/**",
     "test": "run-s test-serve test-build",
-    "test-serve": "jest",
+    "test-serve": "node ./scripts/setup.cjs && uvu packages/playground \"(alias|assets|cli)/__tests__/.*\\.spec\\.[jt]sx?\" -r jiti/register && node ./scripts/teardown.cjs",
     "debug-serve": "cross-env VITE_DEBUG_SERVE=1 node --inspect-brk ./node_modules/.bin/jest",
```

Comment:
still jest

Reply:
This PR avoids the unnecessary changes for initial migration.